examples/playground/getting-started/example-append-array-items/__readme.yaml

```diff
@@ -4,28 +4,23 @@ readme: |
   Appending Array Items
   =====================
 
-  Map items are easy to match: if the key of the "left" and
```

Comment:
this section should also probably change a bit given we are just defaulting by for overlay/match. cc @jtigger you may have some opinion on wording/structure.